test/fixtures.spec.js

```diff
@@ -92,7 +93,78 @@ describe('fixtures', () => {
               };
               break;
 
+            case 'remove-context-wrap-es6':
```

Comment:
Why adding those two type of tests? I don't think that we need those. I would rather use on of the following options:
- Use the `options.json` file (inspiration taken from Babel
- Enable the `contextMode` option for the tests